Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/util/mount/nsenter_mount.go

```diff
@@ -54,7 +54,7 @@ var _ = Interface(&NsenterMounter{})
 const (
 	hostRootFsPath     = "/rootfs"
 	hostProcMountsPath = "/rootfs/proc/mounts"
-	nsenterPath        = "/nsenter"
+	nsenterPath        = "nsenter"
```

Comment:
@liggitt The test image I used in upstream started out as a scratch image with just the kubelet and nsenter and I never fixed it.